apps/api/src/planning-session/planning-session.service.ts

```diff
@@ -211,7 +211,14 @@ export class PlanningSessionService {
     const occupations = planningSession.occupation;
     const groupedBundles = _.groupBy(careActivities, 'bundle_name');
 
-    const headers = ['Activities Bundle'].concat(occupations.map(e => e.displayName));
+    // adding manual sorting after values are fetched as nested sorts are only part of typeorm 0.3.0 onwards
```

Comment:
Nit- In future, we should upgrade our typeorm version

Reply:
There is no official migration released for 0.2.x to 0.3.x; There are unofficial steps and lot of hit and tries. And what I have gathered from the web is a ton of people have faced issues too.; Hence, refrained from doing it at this hour.